### ISSUE TITLE
The option for cooldown reset should be based on the first entry in the sequence

### ISSUE THREAD

**Issue by Telokis** [ollidiemaus/AutoPotion#38]
Since the sequence will always reset to the start, it doesn't make sense to use the cooldown of anything other than the first entry.  
For example, if, as a druid, I have Renewal with 90s cd and then healthstone with 60s cd, the reset condition needs to be `90` and not `60`.

With this behavior, the sequence will only reset if it can be used again.

**Comment by ollidiemaus** [comment]
Yes you are right I’ll add this as soon as I find time. Am 25.02.2024 um 12:07 schrieb Telokis ***@***.***>:﻿
Since the sequence will always reset to the start, it doesn't make sense to use the cooldown of anything other than the first entry.
For example, if, as a druid, I have Renewal with 90s cd and then healthstone with 60s cd, the reset condition needs to be 90 and not 60.

**Comment by Telokis** [comment]
> Yes you are right I’ll add this as soon as I find time.

@ollidiemaus Thanks for answering so quickly! I can do it myself, no problem!  
From what I can see, it's inside the `buildSpellMacroString()` function. Is `ham.spellIDs` sorted so that `ham.spellIDs[0]` is the first element of the sequence? If so, I have enough information to do it.

**Comment by Telokis** [comment]
@ollidiemaus After taking a closer look, it seems like `ham.spellIDs` only contains info about the user spells. I think there isn't a healthstone or healing pot in there, is that right? If so, the lowest cooldown will always be a self-healing spell which will be placed at the beginning of the sequence.  
If those assumptions are right, this issue is already fixed but we need to update the wording of the config option to make it clearer what it really does.
